src/ACEtk/interpretation/DEDX1/test/dedx1.test.cpp

```diff
@@ -0,0 +1,260 @@
+#define CATCH_CONFIG_MAIN
+
+#include "catch.hpp"
+#include "ACEtk.hpp"
+
+SCENARIO("test interpretation::DEDX1"){
+  
+  using namespace dimwits;
+
+  constexpr auto mev = mega(electronVolt);
+  constexpr auto cm = centi(meter);
+  constexpr auto cc = cm*cm*cm;  
+
+  auto make_approx = [](auto tol){
+    return ranges::view::repeat_n(Approx(0.0).margin(tol), 4);
+  };
+  
+  auto rel_diff = [](auto&& first, auto&& second){
+    return (first-second)/first; 
+  };
+  
+  GIVEN("A faux stopping power data table"){
+    auto table = njoy::ACEtk::Table(njoy::utility::slurpFileToMemory("faux.dedx.data"));
+    const auto faux = njoy::ACEtk::interpretation::DEDX1(std::move(table));
+    
+    WHEN("querying for the target atomic number"){
+      REQUIRE( faux.atomicNumber() == 1 );
+    }
+
+    WHEN("querying for the energy spacing"){
+      REQUIRE( faux.energySpacing() == 0.0 );
+    }
+
+    WHEN("querying for the density spacing"){
+      REQUIRE( faux.densitySpacing() == 0.0 );
+    }
+
+    WHEN("querying for the temperature spacing"){
+      REQUIRE( faux.temperatureSpacing() == 0.0 );
+    }
+
+    WHEN("querying for standard stopping power model without a cutoff"){
+      const auto standard = faux.standardWithoutCutoff();      
+      auto sp = standard.stoppingPowers();
+      auto base = std::vector<double>(2,0.0);
+      double i = 0.0;
+      auto increment = [&](auto&& val){return val+i;};	
+
+      AND_THEN("iterating through the stopping power values and "
+	       "querying for the values"){
+
+	RANGES_FOR(auto&& entry, sp){
+	  auto reference = base | ranges::view::transform(increment);
```

Comment:
the intent is that this is a repeated value, right? prefer `ranges::view::repeat_n`